index.js

```diff
@@ -6,47 +6,58 @@ function subtract(a,b){
   return a-b
 }
 
-function verifyInput(number1,number2){
-  if((typeof number1 !== 'number')||(typeof number2 !== 'number')){
+function verifyInput(num1,num2){
+  if ((Number.isInteger ( num1 )) && (Number.isInteger ( num2 ))){
```

Comment:
you add this check, to see if the number is an integer, but you did not add any test to it, can you add a test with float numbers, to make this check useful ?

Reply:
I have added below test. Please reconfirm the change.
```suggestion
  expect(verifyInput(20,8.5)).toBe(false)
```